src/test/java/com/qunite/api/web/QueueControllerTest.java

```diff
@@ -199,14 +173,13 @@ void createQueue() throws Exception {
     var json = new ObjectMapper().writeValueAsString(dto);
 
     given(queueService.create(any(Queue.class))).willReturn(queue);
-    var resultActions = mockMvc.perform(post(url + "/")
+    var resultActions = mockMvc.perform(post(url)
         .contentType(MediaType.APPLICATION_JSON)
         .content(json));
     resultActions.andExpect(status().isCreated())
-        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaTypes.JSON_API_VALUE))
+        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
         .andExpect(content().string(allOf(
-            containsString("\"id\":\"1\""),
-            containsString("\"type\":\"queues\"")
+            containsString("\"id\":1,")
```

Comment:
I do not understand the reason for this decision. You can check for type and for the presence of a field and it will look better